OracleSOASuite/dockerfiles/12.2.1.3/Dockerfile.patch

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at
+# http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle SOA Suite
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# See soasuite.download and patches.download files in the install directory
+#
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.3
```

Comment:
Why isn't this based off the more recent `oracle/fmw-infrastructure:12.2.1.4`?

Reply:
This is specific to 12.2.1.3 early access for SOA customers. For 12.2.1.4 we will be pushing a separate Docker file.